src/logicblocks/event/processing/consumers/subscription.py

```diff
@@ -19,7 +19,7 @@ def make_subscriber(
     *,
     subscriber_group: str,
     subscriber_id: str = uuid4().hex,
-    subscriber_sequence: EventSourceIdentifier,
+    subscriber_identifier: EventSourceIdentifier,
```

Comment:
I think this should be called `subscription_request` or something. It isn't the identifier that's important, it's that the subscriber is requesting to subscribe to the thing that the identifier represents.